Tasks/Dzmitruk/Task7/Task7_5.py

```diff
@@ -0,0 +1,21 @@
+
+cache_dict = dict()
```

Comment:
this cache will be reused for every function decorated with @cache which is not exactly correct in most scenarios